app/src/main/java/com/github/factotum_sdp/factotum/ui/display/DisplayFragment.kt

```diff
@@ -30,7 +30,9 @@ class DisplayFragment : Fragment() {
         _binding = FragmentDisplayBinding.inflate(inflater, container, false)
 
         // Set up the recycler view with a photo adapter
-        val photoAdapter = PhotoAdapter()
+        val photoAdapter = PhotoAdapter() { storageReference ->
+            shareImage(storageReference, "1234567890")
+        }
```

Comment:
Prop phone number for now, but could be a const to be more clear

Reply:
Renamed it to a constant, thanks for the suggestion !